### PR TITLE
added tests for OpenHoursService

### PR DESCRIPTION
**Pull request by yuraolkhovskyi**
added tests for OpenHoursService

### CODE REVIEW

File: src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java [hunk 1]
```diff
@@ -1,27 +1,31 @@
 package greencity.service.impl;
 
+import static com.sun.javaws.JnlpxArgs.verify;
```
Comment (ZakharSkaletskyi) [inline src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java]: Where you use this library?

File: src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java [hunk 2]
```diff
@@ -149,4 +145,35 @@ public void findAllTest() {
...
+        List<OpeningHours> genericOpeningHours =
+            new ArrayList<>(Arrays.asList(new OpeningHours(), new OpeningHours()));
+
+        when(openHoursRepo.findAllByPlace(any())).thenReturn(genericOpeningHours);
+
+        List<OpeningHours> foundOpeningHours = openHoursService.getOpenHoursByPlace(place);
+
+        assertEquals(genericOpeningHours, foundOpeningHours);
+        Mockito.verify(openHoursRepo, times(1)).findAllByPlace(any());
+    }
```
Comment (nazarvladyka) [inline src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java]: Please consider about removing some blank lines here and in **findAllByPlaceIdTest**(), now it's to much and it`s not good for big files where you will need to scroll all of this lines.
Consider about better grouping lines, for e.x:

Place place = Place.builder().build();
List<OpeningHours> genericOpeningHours = new ArrayList<>(Arrays.asList(new OpeningHours(), new OpeningHours()));
List<OpeningHours> foundOpeningHours = openHoursService.getOpenHoursByPlace(place);

 when(openHoursRepo.findAllByPlace(any())).thenReturn(genericOpeningHours);

 assertEquals(genericOpeningHours, foundOpeningHours);
 Mockito.verify(openHoursRepo, times(1)).findAllByPlace(any());

And at line **154** better to use just Arrays.asList(new OpeningHours(), new OpeningHours()), without new ArrayList()

File: src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java [hunk 3]
```diff
@@ -149,4 +145,35 @@ public void findAllTest() {
 
         assertEquals(genericEntities, foundEntities);
     }
+
+    @Test
+    public void getOpenHoursByPlaceTest() {
+        Place place = Place.builder().build();
```
Comment (GuN41K) [inline src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java]: What about keyword "new"?

Comment (yuraolkhovskyi) [reply src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java]: I think that there is no difference between creating an object using builder and constructor in such case but I refixed this to "new".

File: src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java (hunk 2 above)
Comment (yuraolkhovskyi) [reply src/test/java/greencity/service/impl/OpenHoursServiceImplTest.java]: Actually I've done this in such way for more readability but I got you. And also when I put foundOpeningHours under genericOpeningHours test fails and the same case with set in method findAllByPlaceId.
Btw, thanks for useful hints.

**Pull request merged**